Automattic/jetpack
# Social: Remove media auto-conversion toggle UI

We are getting rid of media auto conversion UI because we want to enable it by defaul.

## Proposed changes:

* Remove the media auto-conversion toggle from Social and Jetpack settings

### Other information:


## Jetpack product discussion



## Does this pull request change what data or activity we track or use?



## Testing instructions:






* Go to Social and Jetpack settings pages
* Confirm that you no longer see the auto conversion toggle

Nuked
<img width="862" alt="image" src="https://github.com/user-attachments/assets/b87acd22-1e15-4e0d-8ac3-c2bb4bab0cd8">
<img width="728" alt="image" src="https://github.com/user-attachments/assets/0c470412-c62f-4a49-97ce-0f5d72bb7096">
